front/src/styles/scss/applications/operationalStudies/_roundTripsModal.scss

```
@@ -72,30 +72,152 @@
 
       .column-wrapper {
         display: flex;
-        align-items: center;
-        gap: 2px;
+        flex-direction: column;
+        gap: 17px;
         width: 100%;
         padding: 16px;
         background-color: var(--white100);
         border: 1px solid var(--grey10);
         border-radius: 12px;
 
-        // TODO : replace this by card styles
-        div {
-          width: 100%;
+        .round-trip-pair {
+          display: flex;
+          align-items: center;
+          gap: 2px;
+
+          .separator {
+            width: 4px;
+            height: 38px;
+            border-radius: 2px;
+            background-color: var(--grey5);
+
+            &.valid {
+              background-color: var(--info30);
+            }
+            &.invalid {
+              background-color: var(--warning30);
+            }
+          }
         }
 
-        .card-placeholder {
-          height: 67px;
-          background-color: var(--grey5);
+        .round-trips-card {
+          display: flex;
+          flex-direction: column;
+          flex: 1;
+          min-width: 0;
+          min-height: 67px;
           border-radius: 8px;
+          border: 1px solid var(--grey30);
+
+          .round-trips-card-header {
+            display: flex;
+            align-items: center;
+            height: 26px;
+            color: var(--black100);
+            background-color: var(--white100);
+            border-top-left-radius: 8px;
+            border-top-right-radius: 8px;
+            border-bottom: 1px solid var(--grey30);
+
+            .name {
+              flex: 1;
+              font-size: 14px;
+              font-weight: 600;
+              margin-bottom: 0;
+              white-space: nowrap;
+              overflow: hidden;
+              text-overflow: ellipsis;
+              padding-inline: 12px 4px;
+            }
+
+            .interval {
```

I think it misses line-height, it seems a little too upwards
```suggestion
            .interval {
                line-height:16px;
```

---

Tried with it but couldn't see any differences, did you ?

---

hmm i was seeing something, but i retested it and not anymore... let's pretend nothing happened here :)